File: lib/DevOther/src/main/java/kotlin/dev/engine/cache/CacheConfig.java
package kotlin.dev.engine.cache;

import dev.engine.cache.ICacheEngine;
import dev.utils.app.cache.DevCache;
import dev.utils.common.cipher.Cipher;

/**
 * detail: Cache Config
 * @author Ttt
 */
public class CacheConfig
        extends ICacheEngine.EngineConfig {

    public final DevCache mDevCache;

    public CacheConfig(
            String cacheID,
            Cipher cipher,
            DevCache cache
    ) {
        super(cacheID, cipher);
        this.mDevCache = cache;
    }
}

File: lib/DevOther/src/main/java/ktx/dev/engine/cache/CacheConfig.kt
package ktx.dev.engine.cache

import dev.engine.cache.ICacheEngine
import dev.utils.app.cache.DevCache
import dev.utils.common.cipher.Cipher

/**
 * detail: Cache Config
 * @author Ttt
 */
class CacheConfig(
    cipher: Cipher?,
    val mDevCache: DevCache
) : ICacheEngine.EngineConfig(cipher)